### ISSUE TITLE
no matches converting function ‘create_point’

### ISSUE THREAD

**Issue by qwiglydee** [osmcode/pyosmium#6]
I have built and installed libosmium-2.5.2 from sources.
(with all the tests and they all passed)

Building pyosmium-2.4.1 fails:
```
lib/geom.cc:42:126: error: no matches converting function ‘create_point’ to type ‘std::string (class WKBFactory::*)(class osmium::Location)const {aka class std::basic_string<char> (class WKBFactory::*)(class osmium::Location)const}’
         .def("create_point", static_cast<std::string (WKBFactory::*)(const osmium::Location) const>(&WKBFactory::create_point),
                                                                                                                              ^
In file included from lib/geom.cc:4:0:
/usr/local/include/osmium/geom/factory.hpp:215:24: note: candidates are: osmium::geom::GeometryFactory<TGeomImpl, TProjection>::point_type osmium::geom::GeometryFactory<TGeomImpl, TProjection>::create_point(const osmium::NodeRef&) [with TGeomImpl = osmium::geom::detail::WKBFactoryImpl; TProjection = osmium::geom::IdentityProjection; osmium::geom::GeometryFactory<TGeomImpl, TProjection>::point_type = std::basic_string<char>]
             point_type create_point(const osmium::NodeRef& node_ref) {
                        ^
/usr/local/include/osmium/geom/factory.hpp:206:24: note:                 osmium::geom::GeometryFactory<TGeomImpl, TProjection>::point_type osmium::geom::GeometryFactory<TGeomImpl, TProjection>::create_point(const osmium::Node&) [with TGeomImpl = osmium::geom::detail::WKBFactoryImpl; TProjection = osmium::geom::IdentityProjection; osmium::geom::GeometryFactory<TGeomImpl, TProjection>::point_type = std::basic_string<char>]
             point_type create_point(const osmium::Node& node) {
                        ^
/usr/local/include/osmium/geom/factory.hpp:202:24: note:                 osmium::geom::GeometryFactory<TGeomImpl, TProjection>::point_type osmium::geom::GeometryFactory<TGeomImpl, TProjection>::create_point(const osmium::Location&) const [with TGeomImpl = osmium::geom::detail::WKBFactoryImpl; TProjection = osmium::geom::IdentityProjection; osmium::geom::GeometryFactory<TGeomImpl, TProjection>::point_type = std::basic_string<char>]
             point_type create_point(const osmium::Location& location) const {
                        ^
error: command 'x86_64-linux-gnu-gcc' failed with exit status 1
```

**Comment by lonvia** [comment]
This issue has already been fixed in 1138b98ec64c903d2666ed9c8efd83e5c4c11825. A 2.5 release will follow as soon as possible. In the meantime please use the github master branch.

**Comment by qwiglydee** [comment]
oops! I missed version mismatch.
